documentation/smg/10_build_release_process.md

```diff
@@ -41,6 +41,8 @@ Horace PR must not be merged until the Herbert PR has been merged
 5. If the build/test of the PR against the Release branch succeeds on all
 platforms, and the code has been reviewed, the PR can be merged.
 
+<img src="./images/10_developmentworkflow.png">
```

Comment:
1. Being picky, extra step "branch codebase" could be good.
2. Not wanting Simpsons references, not Marge PR